### PR TITLE
Fix paths on eca plots

### PR DESCRIPTION
**Pull request by jcaravaca**
Changed path of the eca plots generated by the eca monitor. ECA script will create different folders for each run in /mnt/eca with the same file name rather than append the run number on the file name. Remove test plots as well.

### CODE REVIEW

File: minard/views.py [hunk 1]
```diff
@@ -467,17 +467,18 @@ def metric():
 
 @app.route('/eca')
 def eca():
+
     runs = ecadb.runs_after_run(0)      
     return render_template('eca.html', runs=runs)
  
-@app.route('/eca_run_detail/<int:run_number>')
```
Comment (tlatorre-uchicago) [inline minard/views.py]: Does this url rule not automatically convert run_number -> int for you?

**Comment by tlatorre-uchicago** [comment]
Did you mean to delete all of the image files in the eca test folder?

File: minard/views.py (hunk 1 above)
Comment (jcaravaca) [inline minard/views.py]: Yes, but render_template('eca_run_detail_%s.html' % run_type, run_number=run_number) appends the run number to a string and doesn't like an integer. I could change the template instead, but I decided rather to remove that conversion.

**Comment by jcaravaca** [comment]
They are not needed anymore as we will copy pngs from real data next week. If you prefer to have them in the repo I can bring them back.

**Pull request merged**